sync/class.jetpack-sync-actions.php

```diff
@@ -135,11 +135,16 @@ static function send_data( $data, $codec_name, $sent_timestamp, $queue_id ) {
 		return $rpc->getResponse();
 	}
 
+	static function get_user_ids_for_initial_sync() {
+		global $wpdb;
+		return $wpdb->get_col( "SELECT user_id FROM $wpdb->usermeta WHERE meta_key = '{$wpdb->base_prefix}user_level' AND meta_value > 0" );
```

Comment:
I think we need to do set a limit here and chunk process it in batches. Like we currently do for users full sync.